packages/browser-wallet-api/README.md

```diff
@@ -0,0 +1,88 @@
+# browser-wallet-api
+
+This package is an implementation of an API to be used in web applications for communicating with the Concordium browser wallet.
+
+## Using the API
+
+The API is automatically injected into web applications if the Concordium browser wallet extension is installed in the browser. It is made accessible `window.concordium`. A callback (`concordiumReady`), which needs to be defined globally, is called by the injected script to notify applications that the API is ready for use. The following exemplifies how this can be done.
```

Comment:
```suggestion
The API is automatically injected into web applications if the Concordium browser wallet extension is installed in the browser. It is made accessible by `window.concordium`. A callback (`concordiumReady`), which needs to be defined globally, is called by the injected script to notify applications that the API is ready for use. The following exemplifies how this can be done.
```